MAUI/Maps/tooltip.md

```diff
@@ -445,8 +555,13 @@ Data template can be used customize the tooltip view using [`ShapeTooltipTemplat
 ![Maps tooltip appearance customization](images/tooltip/tooltip_textStyle.png)
 
 N>
-* The BindingContext of the Shape and Bubble will be the corresponding underline object.
-* The BindingContext of the Marker will be the corresponding MapMarker.
+* The `BindingContext` of the Tooltip will be the `MapTooltipInfo`, and it has the `DataItem` property. This property holds the corresponding underline object for shape, bubble tooltip and holds the corresponding `MapMarker` for the marker tooltip.
```

Comment:
* The `BindingContext` of the Tooltip will be the `MapTooltipInfo,` and it has the `DataItem` property. This property holds the corresponding underline object for the shape, bubble tooltip, and the corresponding `MapMarker` for the marker tooltip.

Reply:
I have changed.